src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -2136,6 +2140,10 @@
   <data name="UnitName_Joule" xml:space="preserve">
     <value>Joules</value>
     <comment>A measurement unit for energy. (Please choose the most appropriate plural form to fit any number between 0 and 999,999,999,999,999)</comment>
+  </data>
+    <data name="UnitName_Kilowatthour" xml:space="preserve">
+    <value>Kilowatt-hours</value>
```

Comment:
I suppose it might be `Kilowatt hours`. The `Foot-pounds` may not be a good example here and we need to review the name again in future to see if it is a correct name.

Reply:
That Wikipedia article's Talk page is instructive.

From the debates there, it seems as if common usage is to write "Kilowatt-hours" and "kWh".
Strict technical usage, as seen in IEC and NIST documents, is to write "Kilowatt hours" and "kW·h".

We certainly haven't adopted the strictly-correct abbreviation "kW·h" (for good reason – no one really uses this), so I'd say we should also stick with common usage of the long name: "Kilowatt-hours".